src/plugins/irc-events/sasl.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+const Msg = require("../../models/msg");
+
+module.exports = function (irc, network) {
+	const client = this;
+
+	irc.on("loggedin", (data) => {
+		const lobby = network.channels[0];
+
+		const msg = new Msg({
+			type: Msg.Type.AUTH,
+			text: "Logged in as: " + data.account,
+		});
+		lobby.pushMessage(client, msg, false);
```

Comment:
Code looks good, but I'm wondering if we want this to be `true` (for those reading here, `increasesUnread`)

Reply:
I set this to `false` because I don't think it's worth reading, but I also think that about MOTD and the like (which is `true`), so it's your call

Reply:
Yeah, for explicitness's sake we should increase the unread to decrease potential confusion. New message in window === increase the unread, generally. For example, I want to know if I'm logged out by the network. We already spam the user with messages on connection, so one more about logging in is nbd imo

Reply:
done